src/panels/lovelace/cards/hui-calendar-card.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+
+import "../../../components/ha-card";
+import "../../../components/ha-icon";
+import "../../calendar/ha-full-calendar";
+
+import type {
+  HomeAssistant,
+  CalendarEvent,
+  Calendar,
+  CalendarViewChanged,
+  FullCalendarView,
+} from "../../../types";
+import type { LovelaceCard, LovelaceCardEditor } from "../types";
+import type { CalendarCardConfig } from "./types";
+import { findEntities } from "../common/find-entites";
+import { applyThemesOnElement } from "../../../common/dom/apply_themes_on_element";
+import "../components/hui-warning";
+import { fetchCalendarEvents } from "../../../data/calendar";
+import { HASSDomEvent } from "../../../common/dom/fire_event";
+import { HA_COLOR_PALETTE } from "../../../common/const";
+
+@customElement("hui-calendar-card")
+export class HuiCalendarCard extends LitElement implements LovelaceCard {
+  public static async getConfigElement(): Promise<LovelaceCardEditor> {
+    await import(
+      /* webpackChunkName: "hui-calendar-card-editor" */ "../editor/config-elements/hui-calendar-card-editor"
+    );
+    return document.createElement("hui-calendar-card-editor");
+  }
+
+  public static getStubConfig(
+    hass: HomeAssistant,
+    entities: string[],
+    entitiesFill: string[]
+  ) {
+    const includeDomains = ["calendar"];
+    const maxEntities = 2;
+    const foundEntities = findEntities(
+      hass,
+      maxEntities,
+      entities,
+      entitiesFill,
+      includeDomains
+    );
+
+    return {
+      entities: foundEntities,
+    };
+  }
+
+  @property() public hass?: HomeAssistant;
+
+  @property() private _config?: CalendarCardConfig;
+
+  @property() public _events: CalendarEvent[] = [];
+
+  @property() private _calendars: Calendar[] = [];
```

Comment:
Use `internalProperty` for private properties.
```suggestion
  @internalProperty() private _calendars: Calendar[] = [];
```